crowbar_framework/config/routes.d/api_v2_storage.routes

```diff
@@ -0,0 +1,11 @@
+namespace :api,
```

Comment:
I would name the file just `storage.routes` because we will have all storage related routes here.